README.md

```diff
@@ -184,43 +90,6 @@ All imports must import components/containers under their original name. (So onc
 
 ```
 
-
-<a name="react-methods-props"></a>
-### [Method and property naming in React](#react-methods-props)
```

Comment:
I would keep this (at least a part of it) with naming convention `on*` and `handle*` - and/or rewrite example using hooks

Reply:
ok, adjusted the naming-part with a hook-based approach.